src/routing.rs

```diff
@@ -190,6 +175,7 @@ pub fn push_route<U: Into<Url>>(url: U) -> Url {
 
     util::history()
         .push_state_with_url(&data, title, Some(&path))
+//        .push_state_with_url(&url.to_str(), title, Some(&path))
```

Comment:
Forgotten comment?

Reply:
Yea, was experimenting with your suggestion from the Issue. Unrelated: I'm getting the async CI failure on my machine as well; reverting Cargo.toml for now.

Reply:
> was experimenting with your suggestion from the Issue

Don't use `data` and use current url instead?